Product.md

```diff
@@ -0,0 +1,31 @@
+# Product Roles
+
+## Product Manager
+
+> I understand and advocate for Octopus users, and have a deep knowledge of the ecosystem Octopus lives in.  I help shape the strategic direction for the Octopus product. I am passionate about building software that users love. 
+
+### I understand and advocate for Octopus users
+
+- I spend time listening to users, through various communication channels  
+- I take the time to understand the problems they are trying to solve 
+- I use data effectively to predict and report the success of delivered features 
+- I use the insight I gain about our users to improve the Octopus product 
+
+### I understand the ecosystem Octopus lives in 
+
+- I have a deep working knowledge of the software development lifecycle 
+- I spend time evaluating other tools in the ecosystem
+- I understand Octopus's relative strengths and opportunities
+- I use the insight I gain about our competitors to improve the Octopus product 
+
+### I am recognised for my communication skills   
+
+- I am an information honey bee, helping all the teams at Octopus connect the dots 
+- I communicate clearly, and I communicate a _lot_
+- I am passionate and persuasive when discussing product direction 
+- I help the marketing team to tell a compelling story
+
+### I play a key role in delivering software that users love 
```

Comment:
Is there value calling out the influence you can have with the development teams around scope hammering? Effective input from a product manager can help the team deliver more valuable things with the right level of lovability.

Reply:
I've also seen a positive influence product managers can have when engineering teams are struggling to make decisions. I liked the decisiveness introduced when considering RHEL6 support, or the script execution priority for each platform, for examples.